lldb/test/API/tools/lldb-dap/breakpoint-events/TestDAP_breakpointEvents.py

```diff
@@ -60,7 +60,7 @@ def test_breakpoint_events(self):
         response = self.dap_server.request_setBreakpoints(
             main_source_path, [main_bp_line]
         )
-        self.assertTrue(response)
+        self.assertTrue(response["success"])
```

Comment:
We could add an `assertResponse` function to the dap test base class to streamline these kind of asserts.